src/routes/class/home/Home.tsx

```diff
@@ -2,17 +2,19 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import { RootState } from 'store/store';
 
-import ClassProgress from 'components/ClassProgress/ClassProgress';
+import ClassProgressCard from 'components/ClassProgressCard/ClassProgressCard';
 import ContinueActivityCard from 'components/ContinueActivityCard/ContinueActivityCard';
 import UpcomingHomeworkCard from 'components/UpcomingHomeworkCard/UpcomingHomeworkCard';
 import HomeLeaderboardPositionCard from 'components/HomeLeaderboardPositionCard/HomeLeaderboardPositionCard';
 
+import ClassModulesProgressCard from 'components/ClassModulesProgressCard/ClassModulesProgressCard';
+
 import styles from './Home.module.css';
 
 function StudentHome() {
   return (
     <section className={styles['home-container']}>
-      <ClassProgress className={styles['home-container-item']} />
+      <ClassProgressCard className={styles['home-container-item']} />
```

Comment:
It would be cool to later on add a skeleton for this Home component

Reply:
I just created an issue to do it :)